frontend/src/components/eMIB/OrganizationalStructure.jsx

```diff
@@ -295,7 +313,6 @@ class OrganizationalStructure extends Component {
               onClick={this.openPopup}
               className="btn btn-secondary"
               style={styles.button}
-              aria-label={LOCALIZE.emibTest.background.organizationalStructure.orgChart.ariaLabel}
```

Comment:
This aria label is needed for accessibility

Reply:
See #434

Reply:
I did not mean to remove it.
Good catch.
Added back in commit #https://github.com/code-for-canada/project-thundercat/pull/437/commits/630dd422573c7399b3bef05fd5f3e372b113e831